deploy/src/main/deploy/docker/swarm_deploy.sh

```diff
@@ -19,14 +19,27 @@ CONFIG=$SCRIPTPATH/../../config
 CERTS=$CONFIG/hono-demo-certs-jar
 NS=hono
 CREATE_OPTIONS="-l project=$NS --network $NS --detach=false"
+if [ -z "$1" ] ; then
+  SLIM=0
+else 
+  SLIM=$1
+fi
 
+if [ $SLIM -ne 1 ]
+then
 echo DEPLOYING ECLIPSE HONO TO DOCKER SWARM
+else
+echo SLIM DEPLOYING ECLIPSE HONO TO DOCKER SWARM
```

Comment:
what's the purpose of this?

Reply:
Commit comment:

> Add option for slim deployment, which omit to deploy the adapters in
order to start an adapter from a IDE.

It makes it easier to develop an adapter. It starts all required parts except the adapter, which is the started separately. Enables faster turn-around during adapter development.